.github/workflows/scripts/free_disk_space.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+#
+# Based on: https://github.com/apache/flink/blob/master/tools/azure-pipelines/free_disk_space.sh
+#
+echo "=============================================================================="
+echo "Freeing up disk space on CI system"
+echo "=============================================================================="
+echo ""
+
+# Before
+echo "Disk space before:"
+df -h
+echo ""
+
+# List packages
+echo "Listing 100 largest packages"
+dpkg-query -Wf '${Installed-Size}\t${Package}\n' | sort -n | tail -n 100
+echo ""
+
+# Remove packages
+echo ""
+echo "Removing large packages"
+sudo apt-get remove -y '^dotnet-.*'
+sudo apt-get remove -y '^llvm-.*'
+sudo apt-get remove -y 'php.*'
+sudo apt-get remove -y '^mongodb-.*'
+sudo apt-get remove -y '^mysql-.*'
+sudo apt-get remove -y azure-cli google-cloud-sdk hhvm google-chrome-stable firefox powershell mono-devel
+sudo apt-get autoremove -y
+sudo apt-get clean
+echo ""
+echo "Disk space after apt-get:"
+df -h
+echo ""
+
+# Large dirs
+echo "Removing large directories"
+# https://github.com/apache/flink/blob/master/tools/azure-pipelines/free_disk_space.sh
+sudo rm -rf /usr/share/dotnet/
```

Comment:
```suggestion
```

We already do this.

Reply:
Right, besides dotnet the GitHub Action that runs before this script was also removing Android and Haskell so I removed those steps from the script.